packages/playground/_stories/main/Card/Card.stories.ts

```diff
@@ -47,19 +53,6 @@ const header = (titleText: string, subtitleText: string, status?: string, action
 </ui5-card-header>`;
 };
 
-const setWidth = (width: string) => {
-	const style = html`
-		<style>
-			ui5-card {
-				width: ${width};
-			}
-		</style>`;
-
-	return (story: () => unknown) => html`
-		${style}
-		${story()}`;
-};
-
```

Comment:
Why remove the decorator and inline the CSS directly?

Reply:
Dur to the reported bug in the table card.

Reply:
The reported bug should be inspected further and not be solved like this (and from us). Looks like the decorators are not respected when all of the samples are rendered in the Docs page. I don't agree with this solution.

Reply:
The decorators for each story should be made to work correctly in every separate sample. This is the best way to solve this.

Reply:
We can keep this fix for now, to create issue for the Story book and when it is solved to revert the fix?

Reply:
If you want to fix it like this and open an issue, then I propose to keep the above code that you are removing and instead render the samples in the Docs page in iframes. This will be the more clean approach. Keeps the HTML in the Stories clean and still uses the Decorators to apply CSS which is the idiomatic approach: https://storybook.js.org/docs/7.1/web-components/writing-stories/decorators